components/Flashcard/Flashcard.js

```diff
@@ -1,30 +1,100 @@
 import styled from "styled-components";
+import { useState } from "react";
 
 export default function Flashcard({ question, answer, collection }) {
+  const [showAnswer, setShowAnswer] = useState(false);
+
+  function handleShowAnswer() {
+    setShowAnswer(!showAnswer);
+  }
+
   return (
-    <StyledFlashcard>
-      <CollectionTitle>{collection}</CollectionTitle>
-      <Question>{question}</Question>
-      <p>{answer}</p>
-    </StyledFlashcard>
+    <CardContainer onClick={handleShowAnswer}>
+      <StyledFlashcard $showAnswer={showAnswer}>
+        <CardFront>
+          <CollectionTitle>{collection}</CollectionTitle>
+          <Question>{question}</Question>
+          <StyledArrow
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 -960 960 960"
+            width="24px"
+            fill="#000"
+          >
+            <path d="m700-300-57-56 84-84H120v-80h607l-83-84 57-56 179 180-180 180Z" />
+          </StyledArrow>
```

Comment:
I suggest to move the `StyledArrow` outside the `CardFront` and `CardBack` to avoid redundancy.

```         
<CardFront>
    <CollectionTitle>{collection}</CollectionTitle>
    <Question>{question}</Question>
</CardFront>
<CardBack>
    <Answer>{answer}</Answer>
</CardBack>
<StyledArrow
          xmlns="http://www.w3.org/2000/svg"
          height="24px"
          viewBox="0 -960 960 960"
          width="24px"
          fill="#000"
          transform={showAnswer ? "scale(-1 1)" : "none"}
>
    <path d="m700-300-57-56 84-84H120v-80h607l-83-84 57-56 179 180-180 180Z" />
</StyledArrow>
```

Reply:
If we do it as you suggested, the arrow is no longer part of the smooth animation, but rotates directly when the useState changes. We actually thought your suggestion was a good hint, but in practice it looks like it will be more complicated than before, so we'll leave it as it was.
We tried this, among other things:

```
const StyledArrow = styled.svg`
  position: absolute;
  ${({ $showAnswer }) => ($showAnswer ? "left" : "right")} : 10px;
  bottom: 10px;
`;
```